cmd/logmonitor/db.go

```diff
@@ -0,0 +1,147 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
Would you mind moving the database operations into their own go package with their own unit tests? 

`cmd` is reserved for `main()` functions which should then use libraries defined elsewhere. 
This helps immensely with portability.